casino_classes.py

```diff
@@ -0,0 +1,61 @@
+cardsInDeck = [suit + str(val) for suit in 'cdhs' for val in [i for i in range(2, 11)] + ['j', 'q', 'k', 'a']]
+
+class Hand(object):
+    def __init__(self):
+        self._cards = {}
+        self._handSize = len(self._cards)
+
+    def getCards(self, imageOnly=False):
+        assert type(imageOnly) == bool
+        try:
+            if imageOnly:
+                return self._cards.values()
+            else:
+                return self._cards
+        except AssertionError:
+            pass
+            # invalid request operation
+
+    def getHandSize(self):
+        return self._handSize
+
+    def setCards(self, cards):
+        # create assertion for cards
+        self._cards = cards
+
+    def receiveCard(self, card):
+        # create assertion for card
+        self._cards.update(card)
+
+    def discard(self, cardChoice = None):
+        """
+        discards a card from hand, to discard entire hand perform while self.getHandize > 0 self.discard()
+
+        cardChoice: string (optional)
+            assumes a string in cardsInDeck
+
+        return: dictionary
+            discarded card
+        """
+        assert self.getHandSize() > 0
+        try:
+            if cardChoice is None:
+                return {list(self.getCards())[-1]: self._cards.pop(list(self.getCards())[-1])}
+            else:
+                if cardChoice.lower() in self.getCards():
+                    return {cardChoice.lower(): self._cards.pop(cardChoice)}
+                else:
+                    raise AssertionError
+        except AssertionError:
+            print('Invalid move!')
+            # invalid move operation
+
+
+class Player(object):
```

Comment:
not sure players should receive a hand on init, should get them on deal, after the game has been initialized.

Reply:
players are initialized with self._hand = Hand() for composition with a default hand of an empty list. Players will wait to be dealt cards from dealer and will grow hand with self._hand.receiveCard()

Reply:
Hand() should be the base class from which we will derive our PokerHand() and BlackJackHand() classes, since each game operates differently.  Only containing what's pretty much similar across all card games.

Reply:
so how to do this?